src/style.css

```diff
@@ -0,0 +1,19 @@
+body {
+  margin: 0;
+}
+
+div {
```

Comment:
- usually you will need a general class for elements with the same styles; styling a tag is not the best choice since you may have other div tags that don't need this style